react-frontend/src/components/Nav/navbar.js

```diff
@@ -131,6 +133,15 @@ function NavBar() {
                 Products & Services
               </NavBarLink>
             </NavBarLi>
+            <NavBarLi>
```

Comment:
Would you like to add another menu item, before Marketplace, for Case Studies? Then I can blow up #443.

Reply:
sure

Reply:
done